astrauli/user-posts-code-challenge
# Queryable UI

### Description
- Implement UI to allow for easier testing of the API

### Test plan
- Ensure DB is running locally - `docker-compose up -d`
- Start server (from `/api` directory) - `npm run dev`
- Start UI (from `/web`) - `npm run dev`

Navigate the UI to test things out
- There are 2 pages
  - `http://localhost:3001/` that allows testing of all 5 users endpoints
  - `http://localhost:3001/posts` that allows testing of all 4 post endpoints

Screenshot of the home interface
<img width="1378" alt="Screenshot 2023-08-14 at 8 16 08 PM" src="https://github.com/astrauli/user-posts-code-challenge/assets/17238622/95459639-b6b0-4be1-b18b-8231521be17a">
